site/content/en/latest/boilerplates/testing.md

```diff
@@ -0,0 +1,46 @@
+---
+
+---
+
+{{< tabpane text=true >}}
```

Comment:
to avoid duplication, can we delete https://github.com/envoyproxy/gateway/blob/d591eb6fba06f9d9b768b32e77e0131f2084d9cf/site/content/en/latest/boilerplates/prerequisites.md?plain=1#L24 and make it a include in that page, so its only defined once  ?